torch/csrc/Size.cpp

```diff
@@ -39,6 +34,7 @@ static PyObject * THPSize_pynew(PyTypeObject *type, PyObject *args, PyObject *kw
 
 static PyObject * THPSize_repr(THPSize *self)
 {
+  HANDLE_TH_ERRORS
```

Comment:
can we rename this macro?  I thought I only needed to guard things when I call into TH, but here we never touch TH.

Reply:
Yeah, that seems like a good idea. (We need the macro around any calls which may throw C++ exceptions)